src/s4c_the_church_rap_battle.tw2

```diff
@@ -0,0 +1,343 @@
+Grab a hold of these buns, they hard steel
+I’m the primo shit talker, real walker//(stop:)]]]]
+[(live: 1s)[(if: time > 11s)[$dissolve[//No texas ranger, but you gettin’ stranger
+I can hear your tired rhymes gettin’ fainter//(stop:)]]]]
+
+[(live: 1s)[(if: time > 13s)[$dissolve[//Why you runnin’ baby? Just took out my knife.
+Better hurry it up, if you value life//(stop:)]]]]
+[(live: 1s)[(if: time > 15s)[$dissolve[//Cause you heard of me, girl I’m Smooth Lips Steve
+If you wanna live, best get on your knees//(stop:)]]]]
+
+[(live: 1s)[(if: time > 17s)[$dissolve[(if: $playerRapsFirst is true)[A dude I thought was Steve’s homie made a huge fart noise with his mouth. It was [[time for results|The Church - Rap battle - Victory player first]].]
+\(else:)[I had to scoff at that lame-ass attempt and a few people noticed. I heard some “ooohs” as I [[went into my response|The Church - Rap battle - Second verses player]].]
+(stop:)]]]]
+
+::The Church - Rap battle - Second verses player
+I could feel my grip on the mic bending the metal as I howled my final verse straight at the heart of this absolute piece of shit and all of the absolute pieces of shit everywhere and forever.
+
+[(live: 1s)[(if: time > 1s)[$dissolve[//I’m the woman in the mirror
+Gripping on your neck//(stop:)]]]]
+[(live: 1s)[(if: time > 3s)[$dissolve[//Already found the artery
+No use flinching to protect//(stop:)]]]]
+
+[(live: 1s)[(if: time > 5s)[$dissolve[//I’m all out of questions and I never had respect
+You on a throne of shit and I ain’t gonna genuflect//(stop:)]]]]
+[(live: 1s)[(if: time > 7s)[$dissolve[//Had the worst dream ever, woke up in terror
+You were spitting dumb shit in my face forever//(stop:)]]]]
+
+[(live: 1s)[(if: time > 9s)[$dissolve[//Shut the fuck up homie, you don’t know me
+I’m the absolute number one and only//(stop:)]]]]
+[(live: 1s)[(if: time > 11s)[$dissolve[//I guess I could use your head next time I go bowling
+That’s the only way you roll with me//(stop:)]]]]
+[(live: 1s)[(if: time > 13s)[$dissolve[//Otherwise you stay lonely
+Fuck that, I’d never let Steve blow me//(stop:)]]]]
+[(live: 1s)[(if: time > 15s)[$dissolve[//I don’t want leftovers, Fuck you show pony
+All I wanna see is your back now show me//(stop:)]]]]
+
+[(live: 1s)[(if: time > 17s)[$dissolve[(if: $steveRapsFirst is true)[I could feel the power coursing through me and it matched the audience’s roar of approval. [[What a perfect time for the results|The Church - Rap battle - Victory player second]].]
+\(else:)[I could feel the power coursing through me and it matched the audience’s roar of approval. [[Time for Steve’s verse. Ugh|The Church - Rap battle - Second verses steve]].]
+(stop:)]]]]
+
+::The Church - Rap battle - Second verses dogs
+I had to respond to what was in my heart, and what was moving the crowd, so I kept rapping about dogs.
+
+[(live: 1s)[(if: time > 1s)[$dissolve[//I wish I had a puppy when I was growing up
+I’d love to have a flopsy, friendly lil mutt//(stop:)]]]]
```

Comment:
Should be "li'l"

Reply:
Tried to look up the correct version of this and found the most odious spelling forum arguments of all time. I would fight for "Lil" with no apostrophes as the beginning of a rapper name, but in this context your version makes sense.